webview.h

```diff
@@ -922,27 +924,43 @@ class edge_html : public browser {
   std::string init_js = "";
 };
 
+std::string to_lpstr(const LPWSTR s) {
+  int n = WideCharToMultiByte(CP_UTF8, 0, s, -1, NULL, 0, NULL, NULL);
+  std::string result;
+  result.resize(n);
+  WideCharToMultiByte(CP_UTF8, 0, s, -1, result.data(), n, NULL, NULL);
+  return result;
+}
+std::wstring to_lpwstr(const std::string s) {
+  int n = MultiByteToWideChar(CP_UTF8, 0, s.c_str(), -1, NULL, 0);
+  std::wstring result;
+  result.resize(n);
+  MultiByteToWideChar(CP_UTF8, 0, s.c_str(), s.size(), result.data(), n);
+  return result;
+}
+
```

Comment:
Should these be private functions as before or is there a reason for moving them?

Reply:
They used in `edge_chromium` class before but now they used also in `win32_edge_engine` class